getting-started/vs-code-integration/share-to-repl.md

```diff
@@ -0,0 +1,57 @@
+---
+title:  Share to Telerik REPL for Blazor
+page_title: Visual Studio Code Share to Telerik REPL
+description: Learn how to share your Blazor code snippets in Telerik REPL for Blazor.
+slug: getting-started-vs-code-share-to-repl
+position: 25
+---
+
+# Share to Telerik REPL
+
+[Progress Telerik REPL for Blazor]((https://www.telerik.com/blazor-ui/repl)) is a browser-based playground for creating, saving, running and sharing Blazor code snippets.
+
+The [Telerik UI for Blazor Productivity Tools for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=TelerikInc.blazortemplatewizard) provides integration with Telerik REPL for Blazor. It allows you to export your code snippet and share the generated URL to social media, community forums or with a team member.
+
+## Get the Extension
+
+To use the **Share to Telerik REPL for Blazor** option, install the Telerik UI for Blazor Visual Studio Code Extension. 
+@[template](/_contentTemplates/common/general-info.md#vs-code-x-download)
+
+## Share a Code Snippet
+
+To share a code snippet to the Telerik REPL for Blazor you have a few options. 
```

Comment:
Names do not need articles (a/the), unless you use them as adjectives (e.g. the REPL tool).

```suggestion
To share a code snippet to Telerik REPL for Blazor you have a few options. 
```